.github/ISSUE_TEMPLATE.md

```diff
@@ -17,25 +17,23 @@ But if you can please provide more information below.
 - [ ] setting an option in provider's web interface 
 - [ ] other: please describe 
 
-### following advanced settings need to be set
+### Server Settings
 
-neccessary manually typed advanced settings:
+Do you know which servers the provider uses?
 
-    smtp-server: *
-    smtp-port: *
-    smtp-security: *
-    imap-server: *
-    imap-port: *
-    imap-security: *
+```
+    smtp-server: smtp.example.com
+    smtp-port: 25/465/587...
+    smtp-security: SSL/STARTTLS/NONE
```

Comment:
- maybe call it `smtp-socket`, this is also the wording used in the .yml

- `NONE` is currently not an accepted value, only `SSL` or `STARTTLS` are

Reply:
We can include socket, but this is the contributor-facing issue template, so I would still leave security in. It's not defining it very well, I know, but when I first read socket I didn't know what's meant. "security" in combination with "SSL" and "STARTTLS" should communicate what we mean.

Reply:
sounds reasonable.